### PR TITLE
Corrected Export Command Placement

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Export `KUBECONFIG` should be placed before any `kubectl` command.

Changes proposed in this pull request:

- corrected export env placement.

**Related issue(s)**

#442

### CODE REVIEW

**Review by PK85 — DISMISSED** [review]

File: docs/contributor/09-10-ui.md [hunk 1]
```diff
@@ -2,62 +2,63 @@
...
+
+2. For clusters different from Kyma (for example, k3d) you need to install the [prerequisites](../../deployments/prerequisites.yaml).
+    ```shell
+    kubectl apply -f deployments/prerequisites.yaml
+    ```
+
+## Run BTP Manager with UI
+Follow the steps below to run BTP Manager with UI:
+
+1. Make sure the `btp-operator` module is disabled and there are no existing BtpOperator custom resources (CRs) and deployments of BTP Manager and the SAP BTP service operator.
```
Comment (IwonaLanger) [inline docs/contributor/09-10-ui.md:20]:
```suggestion
1. Ensure that the SAP BTP Operator module is deleted and verify that there are no existing BtpOperator custom resources (CRs) or deployments of both BTP Manager and the SAP BTP service operator.
```

File: docs/contributor/09-10-ui.md [hunk 2]
```diff
@@ -2,62 +2,63 @@
...
     ```shell
     kubectl get btpoperators -A
     kubectl get deployment -n kyma-system btp-manager-controller-manager
     kubectl get deployment -n kyma-system sap-btp-operator-controller-manager
     ```
-3. Clone the `btp-manager` repository and checkout to the `sm-integration` branch.
+2. Clone the `btp-manager` repository and checkout to the `sm-integration` branch.
```
Comment (IwonaLanger) [inline docs/contributor/09-10-ui.md:26]:
```suggestion
2. Clone the `btp-manager` repository and check out to the `sm-integration` branch.
```

File: docs/contributor/09-10-ui.md [hunk 3]
```diff
@@ -2,62 +2,63 @@
...
+5. Check if BTP Manager deployment is running.
     ```shell
     kubectl get deployment -n kyma-system btp-manager-controller-manager
     ```
     If you encounter the following error during Pod creation due to Warden's admission webhook:
     ```
-    Error creating: admission webhook "validation.webhook.warden.kyma-project.io" denied the request: Pod images europe-docker.pkg.dev/kyma-project/dev/btp-manager:PR-786 validation failed
+    Error creating: admission webhook "validation.webhook.warden.kyma-project.io" denied the request: Pod images europe-docker.pkg.dev/kyma-project/dev/btp-manager:PR-779 validation failed
     ```
     you must scale the BTP Manager deployment to 0 replicas, delete the webhook, and the scale the deployment back to 1 replica.
```
Comment (IwonaLanger) [inline docs/contributor/09-10-ui.md:47]:
```suggestion
    you must scale the BTP Manager deployment to 0 replicas, delete the webhook, and then scale the deployment back to 1 replica.
```

File: docs/contributor/09-10-ui.md [hunk 4]
```diff
@@ -2,62 +2,63 @@
 
 > [!WARNING]
 > This feature is in the experimental stage and is not yet available in the main branch or official releases.
-> Use the latest development image to test the UI: europe-docker.pkg.dev/kyma-project/dev/btp-manager:PR-786
```
[the remark this answers is not in the document]
Comment (ralikio) [reply docs/contributor/09-10-ui.md:5]: That would be out of scope of this PR. Let's handle it in other PRs.

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**